pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/index/reader/ForwardIndexReader.java

```diff
@@ -723,6 +723,27 @@ default void readValuesMV(int[] docIds, int length, int maxNumValuesPerMVEntry,
     }
   }
 
+  /**
+   * Fills the values
+   * @param docIds Array containing the document ids to read
+   * @param length Number of values to read
+   * @param maxNumValuesPerMVEntry Maximum number of values per MV entry
+   * @param values Values to fill
+   * @param context Reader context
+   */
+  default void readValuesMV(int[] docIds, int length, int maxNumValuesPerMVEntry, byte[][][] values, T context) {
+    switch (getStoredType()) {
```

Comment:
Drop the switch for better performance. `getBytesMV()` can throw the exception if not supported.

Reply:
Done